prov/gni/include/gnix_util.h

```diff
@@ -272,6 +273,15 @@ struct gnix_reference {
 #define _gnix_ref_get(ptr) __ref_get(ptr, ref_cnt)
 #define _gnix_ref_put(ptr) __ref_put(ptr, ref_cnt)
 
+/**
+ * Only allow FI_REMOTE_CQ_DATA when FI_RMA_EVENT is also set.
+ *
+ * @return zero if FI_REMOTE_CQ_DATA is not permitted; otherwise one.
+ */
+#define _gnix_allow_fi_remote_cq_data(_flags) (((_flags) & FI_REMOTE_CQ_DATA) \
```

Comment:
Macros should be fully capitalized unless there is a good reason for it otherwise. That's how we distinguish macro from function at a glance.